l10n_ch_hr_payroll/data/hr.salary.rule.category.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
 File: hr.salary.rule.category.xml
-Module: l10n_ch_hr_payroll
+Module: tw_l10n_ch_hr_payroll
```

Comment:
It seems that you renamed the module before generating translation.

Please do a search and replace for all occurence of tw_l10n_ch_hr_payroll by l10n_ch_hr_payroll
especially in po files.